AstroCloud CLI Reference Guide (#294)
astronomer/docs: cloud/cli-reference/astrocloud-deployment-list.md

---
sidebar_label: 'astrocloud deployment list'
title: 'astrocloud deployment list'
id: astrocloud-deployment-list
description: Reference documentation for astrocloud deployment list.
---

## Description

> List all Deployments that you have access to.  

Author: My bad here, it's for your current Workspace. The option below makes it show across all Workspaces

## Usage

```sh
astrocloud deployment list
```

## Options

| Option  | Description                             | Possible Values |
| ------- | --------------------------------------- | --------------- |
| `--all` | Show deployments across all Workspaces. | ``              |

## Examples

```sh
$ astrocloud deployment list --all

> # Shows Deployments from all Workspaces that you're authenticated to

Reviewer: ```suggestion
# Shows Deployments from all Workspaces that you have access to
```
